package.json

```diff
@@ -30,7 +30,14 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
-    "react-draggable2": "^0.5.1"
+    "react": ">=0.14.0-rc1",
+    "react-dom": ">=0.14.0-rc1",
```

Comment:
We already have those dependencies here. I think they don't need to be duplicated. May be we don't need this package.json https://github.com/Cavitt/material-ui/blob/react-0.14-support/docs/package.json ?